user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -58,8 +58,19 @@ Others
 Interface Changes
 =================
 
+OutgoingRequestInterface
+------------------------
+
```

Comment:
Since 4.3 breaks so many other interfaces maybe add a note:
```suggestion
Note: these changes are all backwards-compatible and require no intervention.

```

Reply:
Great!